### DISCUSSION TITLE
Question: Pistons not working?

### DISCUSSION THREAD

**Discussion by FrostBlade5** [GriefPrevention/GriefPrevention#1799] [Q&A] [answered]
### Observed Behavior

When a player places a piston a message is displayed in chat saying: pistons wont move blocks outside of claimed areas. This is obviously because of grief prevention.

In the config.yml i found this setting PistonMovement: CLAIMS_ONLY
What do i change it to?

### Expected Behavior

looking for help

### Reproduction steps

basic settings can be reproduced anytime

### Server version

```shell
1.18.1
```

### GriefPrevention version

```shell
latest version
```

### Configuration

```yaml
default settings
```

### Running without GriefPrevention

- [ ] I attempted running the server without GriefPrevention installed.
- [ ] The problem does _not_ occur when GriefPrevention is removed from the server.

### Running with only GriefPrevention

- [ ] I attempted running only GriefPrevention on the server.
- [ ] The issue still occurs when GriefPrevention is the only plugin running.

### Running on a fresh, clean server installation

- [ ] I attempted testing for the issue on a new server.
- [ ] The issue still occurs on a new server.

### Using unmodified client

- [ ] I attempted testing for the issue with the vanilla client.
- [ ] The issue still occurs when using the vanilla client.

### We appreciate you taking the time to fill out a bug report!

- [X] I searched for similar issues before submitting this bug report.

**Comment by Jikoo** [comment, marked as answer]
https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#pistonmovement

**Reply by FrostBlade5** [reply]
Thanks
